langchain4j-mcp/src/main/java/dev/langchain4j/mcp/client/transport/http/StreamableHttpMcpTransport.java

```diff
@@ -235,6 +238,15 @@ public StreamableHttpMcpTransport.Builder logResponses(boolean logResponses) {
             return this;
         }
 
+        /**
+         * @param logger an alternate {@link Logger} to be used instead of the default one provided by Langchain4J for logging events.
```

Comment:
I think we should rather use the word "traffic" than events.
Yes I know, in the `stdio` transport, we use "events" to refer to incoming traffic, that wasn't a good choice from my side.

"Logging events" could be easily mistaken for handling MCP protocol-level log events (that the MCP server sends to the client over the protocol). "Traffic" more clearly refers to the actual exchange of protocol messages